resolve columns error: invalid CIDR address: --- Test FAIL:

In tests and I guess in general it's very hard to debug when resolvers fail.

For example the following error doesn't indiciate what column and what it contains:

```
resolve columns error: invalid CIDR address: --- FAIL: TestEc2Eips (0.04s)
```

--- a/provider/schema/execution.go
+++ b/provider/schema/execution.go
@@ -341,7 +341,7 @@ func (e *ExecutionData) resolveColumns(ctx context.Context, meta ClientMeta, res
 			}
 			// check if column resolver defined an IgnoreError function, if it does check if ignore should be ignored.
 			if c.IgnoreError == nil || !c.IgnoreError(err) {
-				return err
+				return fmt.Errorf("column %s: %w", c.Name, err)
 			}
 
 			if reflect2.IsNil(c.Default) {

